src/dropin-model.js

```diff
@@ -205,13 +206,27 @@ function getSupportedPaymentOptions(options) {
 
 function isPaymentOptionEnabled(paymentOption, options) {
   var gatewayConfiguration = options.client.getConfiguration().gatewayConfiguration;
+  var applePayEnabled, applePayBrowserSupported;
 
   if (paymentOption === 'card') {
     return gatewayConfiguration.creditCards.supportedCardTypes.length > 0;
   } else if (paymentOption === 'paypal') {
     return gatewayConfiguration.paypalEnabled && Boolean(options.merchantConfiguration.paypal);
   } else if (paymentOption === 'paypalCredit') {
     return gatewayConfiguration.paypalEnabled && Boolean(options.merchantConfiguration.paypalCredit);
+  } else if (paymentOption === 'applePay') {
+    applePayEnabled = gatewayConfiguration.applePay && Boolean(options.merchantConfiguration.applePay);
+    applePayBrowserSupported = window.ApplePaySession && ApplePaySession.canMakePayments(); // eslint-disable-line no-undef
+
+    if (!applePayEnabled) {
+      return false;
+    }
+    if (!applePayBrowserSupported) {
+      console.log('Browser does not support Apple Pay.'); // eslint-disable-line no-console
+      return false;
+    }
+    analytics.sendEvent(options.client, 'apple-pay.available');
```

Comment:
I think this needs to be stubbed in the unit tests.

Reply:
I've just pushed a commit that does this.

Reply:
Do we actually want this event? I think we get the same thing by looking at apple pay events with the `dropin2` integration type.